working-with-your-data/working-with-your-own-data.md

```diff
@@ -98,7 +103,18 @@ unzip -d data/ data/some_array_data.zip
 
 [Go here](https://itsfoss.com/unzip-linux/) for more on the unzipping command.
 
-### Upload data from your own computer
+**If you have a password:**
+
+You can `wget` data still if you need credentials.
+We don't recommend you put your password or any other credentials in the script, so you will want to type the following directly into the Terminal:
+
+```
+wget --user=<USERNAME> --ask=<PASSWORD> <URL>
```

Comment:
```suggestion
wget --user=<USERNAME> --ask-password <URL>
```

Reply:
The problem is entering a password on the command line, instead of in a way that asks you for your credentials interactively, such that your password is stored in a way that someone could get it with the shell history.